com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/model/ConfigurationSettingsStorable.java

```diff
@@ -14,13 +14,23 @@
 
 package com.ca.lsp.cobol.model;
 
-import lombok.Value;
+import lombok.Builder;
+import lombok.Data;
 
 import java.util.List;
 
-@Value
+/**
+ * This class act as a DTO to store the settings information provided by the user and required to
+ * search copybooks on MF and in the copybook folder.
+ */
+@Data
```

Comment:
By purpose of data consistency, it is better to keep this class immutable and remove the builder. If there is a default value required, it is better to create a special static method and return an empty value there.